wemulate/utils/tcconfig.py

```diff
@@ -274,5 +277,7 @@ def remove_parameters(connection_name: str, interface_name: str) -> None:
     Raises:
         WEmulateExecutionError: if the parameters could not be removed from the interface
     """
-    _execute_in_shell(f"tcdel {interface_name} --all")
+    ipv4_delete_command = f"tcdel {interface_name} --all"
+    ipv6_delete_command = ipv4_delete_command + " --ipv6" 
```

Comment:
```suggestion
    ipv6_delete_command = f"tcdel {interface_name} --all --ipv6" 
```
It's fine, but I think it would be more readable this way